tests/models/hf/test_roberta_hf.py

```diff
@@ -0,0 +1,144 @@
+import pytest
+import torch
+from transformers import (
+    PreTrainedModel,
+    PretrainedConfig,
+    PreTrainedTokenizer,
+    RobertaForMaskedLM,
+    RobertaConfig,
+)
+
+from fms.models.roberta import RoBERTa
+from fms.models.hf.roberta.modeling_roberta_hf import (
+    HFAdaptedRoBERTaConfig,
+    HFAdaptedRoBERTaForMaskedLM,
+)
+from fms.testing._internal.hf.model_test_suite import (
+    HFConfigFixtureMixin,
+    HFModelFixtureMixin,
+    HFConfigTestSuite,
+    HFModelEquivalenceTestSuite,
+    HFModelCompileTestSuite,
+)
+from fms.testing._internal.model_test_suite import ModelFixtureMixin
+from fms.models.hf import to_hf_api
+from ..test_roberta import RoBERTaFixtures
+
+
+class HFAdaptedRoBERTaFixtures(
+    ModelFixtureMixin, HFConfigFixtureMixin, HFModelFixtureMixin
+):
+    @pytest.fixture(scope="class", autouse=True)
+    def fms_hf_model(self, model: RoBERTa, fms_hf_config: PretrainedConfig, **kwargs):
+        return to_hf_api(model, **fms_hf_config.to_dict())
+
+    @pytest.fixture(scope="class", autouse=True)
+    def fms_hf_config(
+        self, tokenizer: PreTrainedTokenizer, model: RoBERTa, **kwargs
+    ) -> PretrainedConfig:
+        bos_token_id = (
+            tokenizer.bos_token_id
+            if tokenizer.bos_token_id is not None
+            else tokenizer.eos_token_id
+        )
+        return HFAdaptedRoBERTaConfig.from_fms_config(
+            model.get_config(),
+            eos_token_id=tokenizer.eos_token_id,
+            bos_token_id=bos_token_id,
+        )
+
+    @pytest.fixture(scope="class", autouse=True)
+    def oss_hf_model(
```

Comment:
should this be in roberta.py? it seems like it's the equivalent of `def convert_hf_llama(hf_model: "LlamaForCausalLM") -> LLaMA:`

Reply:
Yes it's basically the same. It can be separated out into a function

Reply:
Oh sorry, this is the opposite, this is going from fms model to hf model, which really only happens during the tests. We could separate it out into a function, but I don't think its that necessary.